index.js

```diff
@@ -167,26 +167,17 @@ function fireChangeEvent(target: ImageCropElement, result: {x: number, y: number
   target.dispatchEvent(new CustomEvent('image-crop-change', {bubbles: true, detail: result}))
 }
 
-export class ImageCropElement extends HTMLElement {
+class ImageCropElement extends HTMLElement {
   image: HTMLImageElement
   box: HTMLElement
-  constructed: boolean
-  minWidth: number
   dragStartX: ?number
   dragStartY: ?number
   startX: number
   startY: number
 
-  constructor() {
-    super()
-    this.minWidth = 10
-  }
-
   connectedCallback() {
-    if (this.constructed) return
-    this.constructed = true
```

Comment:
@muan I just removed this, not sure what this was doing 😬

Reply:
This was preventing the template from being appended every time `connectedCallback` is triggered since we can't add nodes to the element in constructor.

This can potentially check for existence of `.crop-wrapper` within the node instead. However we should probably also convert these classes to data attributes and update the CSS accordingly since these classes could conflict with the host app.

Reply:
> This was preventing the template from being appended every time `connectedCallback` is triggered since we can't add nodes to the element in constructor.

Cool! I just pushed a commit that checks a `WeakMap` for the element and returns if it has added the template already.

> This can potentially check for existence of `.crop-wrapper` within the node instead. However we should probably also convert these classes to data attributes and update the CSS accordingly since these classes could conflict with the host app.

Would it be cool if I did this in a follow up?